reference/forms/types/integer.rst

```diff
@@ -47,37 +41,6 @@ Field Options
 
 .. include:: /reference/forms/types/options/grouping.rst.inc
 
-.. include:: /reference/forms/types/options/scale.rst.inc
-
-rounding_mode
```

Comment:
Are you sure that the `rounding_mode` option is not used?

Reply:
Yes, rounding_mode option is also hardcoded to `ROUND_DOWN` for `InterType`. (https://github.com/symfony/symfony/blob/a90cd13fa412a082bee8eda1bab2a5bcc8f18cb7/src/Symfony/Component/Form/Extension/Core/DataTransformer/IntegerToLocalizedStringTransformer.php#L32)

Reply:
That's only the default value. But inside the form type's `buildForm()` method the option is passed to the transformer and it will be used then.